openpype/tools/sceneinventory/window.py

```diff
@@ -88,7 +88,8 @@ def __init__(self, parent=None):
         view.setColumnWidth(1, 55)   # version
         view.setColumnWidth(2, 55)   # count
         view.setColumnWidth(3, 150)  # family
-        view.setColumnWidth(4, 100)  # namespace
```

Comment:
FYI: Namespace column didn't exist - this was actually setting the `loader` column. I felt the default width for the loader column was slightly too small and most loader names got cut off. So I've set the loader column width to 150.